samples/bluetooth/llpm/boards/nrf54h20dk_nrf54h20_cpuapp.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Application core does not have BLE controller. Disable controller related options.
+CONFIG_BT_CTLR_SDC_LLPM=n
```

Comment:
Is it sufficient to set `BT_CTLR=n`?

Reply:
Build system complains if this Kconfig option is enabled becuse the BT_CTRL is not enabled for APP core: https://github.com/nrfconnect/sdk-zephyr/blob/6f6b0e4ac941540618a5933449538408acc85c7b/boards/nordic/nrf54h20dk/Kconfig.defconfig#L11-L14